ADR/ADR028-use-notify-for-users.md

```diff
@@ -0,0 +1,20 @@
+# ADR029: Use GOV.UK Notify for emails sent to users for simple notifications
+
+Date: 2024-03-28
+
+## Status
+
+Accepted
+
+
+## Context
+
+[Submission emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_mailer.rb) and [submission confirmation emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_confirmation_mailer.rb) in runner and the [OTP to verify a change in submission email via admin](https://github.com/alphagov/forms-admin/blob/main/app/mailers/submission_email_mailer.rb) are sent via Notify. Notifications to users relating to account management (such as when they are upgraded from a trial to full user account), will be sent. We currently only use Amazon SES for Auth0 One Time Passcode emails where TLS requirements are higher. This is an addendum to [ADR019](ADR019-user-amazon-ses.md).
```

Comment:
Nit picking but since these are externally viewable can we use `GOV.UK Notify` rather than just `Notify` (I know exactly what you mean). 

`, will be sent.` I presume this sentence should end with `using GOV.UK Notify`?

`emails where TLS requirements are higher` since Auth0 cannot integrate with Notify we never explored the option of using Notify for this and consequently whether not enforcing TLS was an issue, though its a benefit of using SES. It might be worth adding that using GOV.UK Notify was not an option for the Auth0 email flow.

Reply:
NOT nitpicky! I'm not the best at ADR/RFC writing and really appreciate this. Have incorporated your suggestions.